spec/models/loan_spec.rb

```diff
@@ -4,18 +4,70 @@
   let(:member)    { users(:user) }
   let(:volunteer) { users(:volunteer) }
 
-  let(:cart)    { Cart.new(member: member, volunteer: volunteer) }
   let(:carrier) { carriers(:carrier) }
 
   describe '#valid?' do
-    context "with a cart, a carrier, and a due date" do
+    context "with a member, a checkout volunteer, a carrier, and a due date" do
       let(:due_date) { DateTime.now + 1.days }
 
-      subject { described_class.new(cart: cart, carrier: carrier, due_date: due_date) }
+      subject {
+        described_class.new(
+          carrier: carrier,
+          due_date: due_date,
+          member: member,
+          checkout_volunteer: volunteer,
+        )
+      }
 
       it "returns true" do
         expect(subject).to be_valid
       end
     end
   end
+
+  describe 'create' do
+    it "sets the due date to the date specified" do
+      due_date = 20.days.from_now.utc.to_date
+      loan = Loan.create(
+        carrier: carrier,
+        member: member,
+        checkout_volunteer: volunteer,
+        due_date: due_date,
+      )
+
+      expect(loan.due_date).to eq(due_date)
```

Comment:
Struggling a bit to understand what exactly we are testing here, probably I'm missing something 😃